Formula/amber.rb

```diff
@@ -0,0 +1,36 @@
+class Amber < Formula
+  desc "Crystal web framework. Bare metal performance, productivity and happiness"
+  homepage "https://amberframework.org/"
+  url "https://github.com/amberframework/amber/archive/refs/tags/v1.3.2.tar.gz"
+  sha256 "37511d6b4afe308e1943cedeab9114b01d5787d868c23d2c0cc555917a21c830"
+  license "MIT"
+
+  depends_on "crystal"
```

Comment:
This should fix your Linux failure:
```suggestion
  depends_on "crystal"
  uses_from_macos "sqlite"
```
When you apply this suggestion, please squash the change into your existing commit and use the commit message
```
amber 1.3.2 (new formula)
```
Thanks!